client/src/components/Header/Header.js

```diff
@@ -320,6 +322,15 @@ const Header = ({ isDark = false }) => {
     return windowSize
   }
 
+  const setActiveNav = (names) => {
+    for (const name of names) {
+      const regex = new RegExp(name)
+      if (useRouter().pathname.search(regex) === 1) {
+        return { textDecoration: 'underline' }
+      }
+    }
+  }
+
```

Comment:
Jag tycker denna funktionen är lite otydlig och kanske onödigt komplicerad. Den heter `setActiveNav`, men kallas på ifrån `<a/>` taggarnas style property som förväntas "få" en property. Kanske vore tydligare om funktionen hette `getNavItemStyles`.

Ett alternativ är om du istället säger ngt i stil med `<a className={isActiveMenuItem('/my-path') ? 'active' : '}/>` och sen i `isActiveMenuItem` bara kollar om argumentet === useRouter.path. Sen har du en css-klass nånstans som säger `a.active {text-decoration: ...}`, alternativt att du gör en styled component av `<a/>`

Reply:
Gillar det senare förslaget, så testar det!

Reply:
Det var knepigt att lösa detta med att classes istället för att addera en style. Fick det inte att funka när det kom till submenyn med `/mpv`, `/scaleup` och `/iteamX` i relation till "Erbjudande", så valde att döpa om funktionen.

Reply:
👍  Gött.

En sista kommentar angående semantic commits.

`chore` är en ändring som inte påverkar "production code", så den hör ffa till ändringar i config och annat kringliggande.

`refactor` är en mer passande flagga för de commitsen du gjorde efter denna tråden skapades.. :)


Referens:
https://www.conventionalcommits.org/en/v1.0.0/#summary
https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716

Reply:
Tack för feedback om detta, vill gärna få in en rutin.